<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainview</class>
 <widget class="QWidget" name="mainview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>565656</width>
    <height>5656565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>中南滴行主界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/贴图库/logo.png</normaloff>:/贴图库/logo.png</iconset>
  </property>
  <widget class="QWidget" name="navigator" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>631</y>
     <width>450</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>160</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button1">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string>广场</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button2">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button3">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string>我的</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="navigate">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>432</width>
     <height>616</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="plaza">
    <widget class="QStackedWidget" name="news_CSU">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>491</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>421</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/贴图库/易);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>易红率团访问韩国、日本</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2024-5-22</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>330</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_8" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>421</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/贴图库/123123.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>毕业生座谈会：从中南出发，拥抱美好未来</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2024-6-6</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>421</width>
         <height>451</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/贴图库/建成.jpg);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>李建成率团访问瑞士、英国</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>2024-6-6</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="nextPageButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>540</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>下一页</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>351</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>请选择您想乘坐的车型：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="economyRadioButton">
        <property name="text">
         <string>实惠快车</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="expressRadioButton">
        <property name="text">
         <string>滴行特快</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="executiveRadioButton">
        <property name="text">
         <string>行政级专车</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="businessRadioButton">
        <property name="text">
         <string>商务六座</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>350</y>
       <width>321</width>
       <height>251</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <widget class="QLabel" name="selectedVehicleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_ride">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>170</width>
        <height>141</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>170</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 25px; /* 这将使按钮成为圆形，确保其宽度和高度相同 */
    background-color:rgb(255, 255, 255);
    min-width: 50px;
    min-height: 50px;
}
QPushButton:pressed {
    background-color: rgb(155, 155, 155);
}
</string>
      </property>
      <property name="text">
       <string>出发！</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="destinationLineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="placeholderText">
      <string>请输入您想到达的目的地...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="user">
    <widget class="QWidget" name="user_info" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>400</width>
       <height>160</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>160</height>
      </size>
     </property>
     <widget class="QLabel" name="name_show">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>76</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>下午好：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>321</x>
        <y>9</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/贴图库/OIP.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>85</y>
        <width>158</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>中南滴行 普通用户</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="vip" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>400</width>
       <height>160</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>160</height>
      </size>
     </property>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>点此输入内部邀请码...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>已是测试版会员？</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vip_bt">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询我的历史订单&gt;&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_orderHistory">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>72</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询10-50s耗时的订单&gt;&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="queryButton">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>72</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="orderListWidget1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>381</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
